src/node_worker.cc

```diff
@@ -142,7 +130,15 @@ Worker::Worker(Environment* env,
   // The new isolate won't be bothered on this thread again.
   isolate_->DiscardThreadSpecificMetadata();
 
-  Debug(this, "Set up worker with id %llu", thread_id_);
+  wrap->Set(env->context(),
+            env->thread_id_string(),
+            Number::New(env->isolate(), static_cast<double>(thread_id_)))
```

Comment:
Now come to think of it, maybe this should be a BigInt?

Reply:
That sounds OK to me